src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The Duke program implements a bot with a set of simple commands
+ * Initializes the app, and interacts with the user.
+ */
+public class Duke {
+
+    /**
+     * The tasks associated with the user
+     **/
+    private TaskList tasks;
+
+    /**
+     * The storage location associated with the user
+     **/
+    private Storage storage;
+
+    /**
+     * The ui interacting with the user
+     **/
+    private Ui ui;
+
+    /**
+     * The default path used when using the empty constructor.
+     */
+    private static final String DEFAULT_PATH = "./data/tasks.txt";
+    
+    /**
+     * Initializes the duke program with a given filePath.
+     *
+     * @param filePath The path of the file containing your list of tasks.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+
+    }
+
+    /**
+     * Load tasks, returning the appropriate message.
```

Comment:
I believe there should have empty line between description and parameter section 👍